LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -160,20 +193,29 @@ function editProperties(isSubForm) {
                     success: function(res) {
                         if(res != null) {
                         }
+                        categories[currCategoryID].sort = $('#sort').val();
                     }
-                }),
+                }),);
+            }
+
+            if(visibleChanged){
                 $.ajax({
                     type: 'POST',
                     url: '../api/?a=formEditor/formVisible',
                     data: {visible: $('#visible').val(),
                         categoryID: currCategoryID,
                         CSRFToken: '<!--{$CSRFToken}-->'},
                     success: function(res) {
+                        categories[currCategoryID].visiblev= $('#visible').val();
```

Comment:
Should this be ```visiblev```? I don't think it's being used against after being defined.

Reply:
Not really sure.  The only thing I did with these is wrap them so they only make the API call if they've changed.  `visblev` does seem to be an obvious typo.